lib/apr/views/commerce/commerce_transaction_slack_view.ex

```diff
@@ -2,9 +2,21 @@ defmodule Apr.Views.CommerceTransactionSlackView do
   import Apr.Views.Helper
   alias Apr.Views.CommerceHelper
 
+  alias Apr.Subscriptions.{
+    Subscription
+  }
+
   @payments Application.get_env(:apr, :payments)
 
-  def render(event, _routing_key) do
+  def render(
+        %Subscription{theme: "fraud"},
+        %{"properties" => %{"order" => %{"items_total_cents" => items_total_cents}}},
+        _routing_key
+      )
+      when items_total_cents < 3000_00,
```

Comment:
superminor but this format is different from above so it looks a little odd.

```suggestion
      when items_total_cents < 300_000,
```